messages/observations/ground_velocity.json

```diff
@@ -5,7 +5,6 @@
     "description": "Reading from a sensor providing information about the ground velocity",
     "required": [
         "sensor_id",
-        "location",
```

Comment:
Me and @TedSjoblom have had a few on-the-side discussion regarding this dwelling on the pros and cons of self-describing messages vs an entity-wide configuration description. I think it would be good to discuss this a bit more in detail.

One possibility is to take advantage of the [retained message](https://www.hivemq.com/blog/mqtt-essentials-part-8-retained-messages/) functionality in MQTT using the broker as a "keeper" of these configurations.